packages/txservice/src/config.ts

```diff
@@ -60,20 +60,26 @@ export const ProviderConfigSchema = Type.Object({
 export type ProviderConfig = Static<typeof ProviderConfigSchema>;
 export const validateProviderConfig = ajv.compile(ProviderConfigSchema);
 
+// TODO: Since moving to exposing the txservice config through the "all" entry in the chainConfig
+// for router json configuration file, we should converge on a single config schema.
+// For example, imagine gasLimitInflation being able to be applied to "all" (and overriden through
+// specification per chain).
+
 /// CHAIN CONFIG
 export const ChainConfigSchema = Type.Object({
   // List of configurations for providers for this chain.
   providers: Type.Array(ProviderConfigSchema),
-  // Quorum is consensus count that must be reached among providers.
-  quorum: Type.Optional(Type.Integer()),
 
   // Hardcoded initial value for gas. This shouldn't be used normally - only temporarily
   // in the event that a gas station is malfunctioning.
   defaultInitialGas: Type.Optional(TIntegerString),
   // Gas station URL, if any, to retrieve current gas price from. If gas station is down or otherwise fails,
   // we'll use the RPC provider's gas as a backup.
-  // Gas station should return a "rapid" gas price within the response.data. See ethereum gasnow API as example.
-  gasStations: Type.Array(Type.String()),
+  // Gas station should return a "rapid" gas price within the response.data.
+  gasStations: Type.Optional(Type.Array(Type.String())),
+  // An integer value by which we will inflate the gas LIMIT that is returned by the provider (flat increase).
+  // Use this if your provider is returning low values and you're getting "out of gas" call exceptions.
+  gasLimitInflation: Type.Optional(Type.Integer()),
```

Comment:
Flat increase in GWEI? or a percentage increase as a buffer?

Reply:
yo great q

Reply:
this is a flat increase in gas lol. gwei is gas price, this is for gas amount.

for example, if we want to inflate by 10_000_000 on arbitrum

Reply:
not related to gas price

and in fact i'm going to change "defaultInitialGas" above to reflect that